pkg/controller/instance/instance_controller.go

```diff
@@ -129,6 +130,15 @@ func (r *ReconcileInstance) Reconcile(request reconcile.Request) (reconcile.Resu
 		}
 	}
 
+	if harbor.DeletionTimestamp != nil {
+		now := metav1.Now()
+		harbor.Status.Phase = registriesv1alpha1.InstanceStatusPhase{
+			Name:           registriesv1alpha1.InstanceStatusPhaseTerminating,
+			Message:        "Deleted",
+			LastTransition: &now}
+		return r.patchInstance(ctx, originalInstance, harbor)
+	}
```

Comment:
When do we set the finalizer on the object? I think the original thought may have been that there's no need to explicitly reconcile the "deleted" status if an object had not been properly created in the first place.